ext/reflection/php_reflection.stub.php

```diff
@@ -665,3 +669,24 @@ private function __clone(): void {}
 
     private function __construct() {}
 }
+
+final class ReflectionEnum extends ReflectionClass
+{
+    public function hasCase(string $name): bool {}
+
+    public function getCase(string $name): ReflectionEnumCase|false {}
+
+    /** @return ReflectionEnumCase[] */
+    public function getCases(): array {}
+
+    public function hasPrimitiveType(): bool {}
+
+    public function getPrimitiveType(): ReflectionType|null {}
+}
+
+final class ReflectionEnumCase extends ReflectionClassConstant
```

Comment:
Would it make sense to override ReflectionClassConstant::getValue() with `getValue(): object` (or `getValue(): UnitEnum` - it's final so the return type can safely be changed if more abstract data types are added later.

(and override the implementation, probably)

This would be a bit more convenient for IDEs/static analyzers.

Also, the RFC could mention that getValue() returns the enum object, and does the same thing as getValue does on a ReflectionClassConstant, since there was a question about that